ebpfapi/rpc_client.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include <ctype.h>
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+#include <windows.h>
+#include "ebpf_api.h"
+#pragma warning(push)
+#pragma warning(disable : 4100) // 'identifier' : unreferenced formal parameter
+#pragma warning(disable : 4244) // 'conversion' conversion from 'type1' to
+                                // 'type2', possible loss of data
+#undef VOID
+#pragma warning(pop)
```

Comment:
This push/pop doesn't wrap any include file, can lines 10-13 and 15 all be deleted or do they serve some purpose?

Reply:
Deleted these lines.